cider/util/memory/allocators/ArenaAllocator.h

```diff
@@ -117,7 +117,9 @@ class Arena {
 
   virtual void* allocate(size_t num_bytes) {
     if (size_limit_ != 0 && size_ + num_bytes > size_limit_) {
-      throw OutOfHostMemory(num_bytes);
+      CIDER_THROW(
+          CiderOutOfMemoryException,
+          fmt::format("Not enough memory available to allocate {} bytes", num_bytes));
```

Comment:
Let's provide more info including size limit and current allocated size.

Reply:
fix